_includes/badge-ui-toolkit.html

```diff
@@ -0,0 +1,3 @@
+<span class="badge badge-ui-toolkit">
+    JS<span class="sr-only"> is required for use of this pattern</span>
```

Comment:
Thanks for both of those thoughts. I'm syncing this up so that we use some native HTML semantics (with the ``<abbr>`` elem) and will pull some of that official badge's characteristics into the ui-toolkit indicators. :smile: